config/config.go

```diff
@@ -113,6 +113,11 @@ type Config struct {
 		Password string `password:"true" info:"Password for authentication."`
 	}
 
+	Webhook struct {
+		Enable    bool     `public:"true" info:"Enables webhook as a contact method."`
+		AllowList []string `public:"true" info:"If set, allows webhooks for these domains only."`
```

Comment:
Let's call this `AllowedURLs` following the naming of some of the other config options.